### ISSUE TITLE
pre login page

### ISSUE THREAD

**Issue by eft0stuff** [wpo365/wpo365-login#45]
Exist any option to show a pre-logon page to the user instead of directly go to the Office 365 Login page?, wish to show the main with a "click here" to login -> Office 365 -> Main intranet website.

Thanks in advanced.

**Comment by wpo365** [comment]
Yes, but not in the BASIC edition. The PROFESSIONAL (and PREMIUM) version, however, offer multiple features that you can combine:

- Dual Login [https://www.wpo365.com/redirect-to-login/](https://www.wpo365.com/redirect-to-login/)
- Sign in with Microsoft shortcode [https://www.wpo365.com/authentication-shortcode/](https://www.wpo365.com/authentication-shortcode/)

The latter you can combine with the public homepage [https://www.wpo365.com/public-homepage/](https://www.wpo365.com/public-homepage/)

Hope this helps!

Please also note that you can contact me through the website [https://www.wpo365.com/contact/](https://www.wpo365.com/contact/)

**Comment by trainingcity** [comment]
Just to follow up on this, I don't mind the non logged in guest to see the main homepage, but I want to redirect the newly logged user to a specific start page. Can I do this with the free version and/or the pro/paid version?
